lib/question.js

```diff
@@ -1117,6 +1117,9 @@ module.exports = {
             urls.baseUrl = urlPrefix;
         }
 
+        const workspaceUrl = urlPrefix + '/workspace/' + variant.workspace_id;
+        urls.workspaceUrl = variant.workspace_id ? workspaceUrl : null;
```

Comment:
Refactor for clearer logic + preference for string templates:
```suggestion
        if (variant.workspace_id) {
          urls.workspaceUrl = `${urlPrefix}/workspace/${variant.workspace_id}`;
        }
```